### PR TITLE
release/5.4 -> develop

### PR DESCRIPTION
**Pull request by AliSoftware**
Finishing release 5.4

### CODE REVIEW

File: WooCommerce/build.gradle [hunk 1]
```diff
@@ -48,9 +48,9 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "5.4-rc-1"
+            versionName "5.4"
         }
-        versionCode 162
+        versionCode 169
```
Comment (AliSoftware) [inline WooCommerce/build.gradle:53]: Just letting the `finish_release` lane auto-bump to `163` led to the following [CI failure](https://app.circleci.com/pipelines/github/woocommerce/woocommerce-android/8382/workflows/96d78128-3034-420f-94e1-e73d97265615/jobs/24955):

```
[18:17:44]: Google Api Error: forbidden: APK specifies a version code that has already been used. - APK specifies a version code that has already been used.
```

I think that's related to the fact that some experiments were conducted during Hack Week leading to some internal testing releases (dynamic features) which means that the `versionCode` was bumped outside the usual track, explaining the gap?

File: build.gradle [hunk 2]
```diff
@@ -85,7 +85,7 @@ task installGitHooks(type: Copy) {
 }
 
 ext {
-    fluxCVersion = '436f9b213886b4b8734ae43b9d4cb1d16dfce997'
+    fluxCVersion = '1.6.26'
     daggerVersion = '2.29.1'
```
Comment (AliSoftware) [inline build.gradle:89]: Note: there was a conflict here when creating this PR (me changing `fluxCVersion` while `daggerVersion` was changed in `develop`), so might be worth double-checking that this conflict resolution is correct

Comment (mokagio) [reply build.gradle:89]: This looks good to me. `1.6.26` is the latest version of the library and the code builds.

File: WooCommerce/build.gradle (hunk 1 above)
Comment (mokagio) [reply WooCommerce/build.gradle:53]: Makes sense 👍 Would be interesting to track down to be sure 🕵️ , but at the same time, this is just an ever increasing number, so as long as it works and is consistent (both of which are ensured by Google) it's fine, I guess 😄

**Review by mokagio — APPROVED** [review]
LGTM. 

Leaving it up to you whether to merge now or get dev with more context on the expected behavior of fluxC and Dagger to chip in regarding the conflicts resolution. I think this is good to go.

**Pull request merged**